BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -592,12 +598,13 @@ def gen_container_bin (container_list, out_dir, inp_dir, key_dir = '.', tool_dir
         out_file = container.create (each)
         print ("Container '%s' was created successfully at:  \n  %s" % (container.header.signature.decode(), out_file))
 
-def gen_layout (comp_list, img_type, out_file, key_file):
+def gen_layout (comp_list, img_type, sign_hash_alg, out_file, key_file):

```

Comment:
you need to pass in both key_dir and key_file

Reply:
I think we could remove key Dir path options and allow user to pass only complete file path.
Incase of Key Dir path, we would assume a TestSigningPrivateKey.pem. It would be better to remove harcoded key file names.